rancher2/data_source_rancher2_cluster.go

```diff
@@ -38,79 +39,69 @@ func dataSourceRancher2Cluster() *schema.Resource {
 			},
 			"rke_config": {
 				Type:     schema.TypeList,
-				MaxItems: 1,
```

Comment:
SDK-V2 dosen't allow "Configurable attributes" for computed-only fields. 

On that case "rke_config" have Computed: true without having optional so the SDK-V2 Complains about the MaxItems config.  Is the same for the "ConflictsWith". 

    data source rancher2_notifier: webhook_config: ConflictsWith is for configurable attributes,there's nothing to configure on computed-only field

Reply:
Yeah, Computed fields usually refer to non-user configurable fields and are determined by the provider on plan/apply execution https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors. So it wouldn't make sense for a Computed field to have a MaxItems defined because that's not configurable anyway. Sounds like v2 is just enforcing it now.